C/Coin-OR/Bonmin/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+using BinaryBuilder, Pkg
+
+name = "Bonmin"
+version = v"1.8.8"
+
+sources = [
+    GitSource("https://github.com/coin-or/Bonmin.git",
+              "65c56cea1e7c40acd9897a2667c11f91d845bb7b"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Bonmin
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/fix_dllexport.patch
+
+# Remove misleading libtool files
+rm -f ${prefix}/lib/*.la
+update_configure_scripts
+
+# old and custom autoconf
+sed -i s/elf64ppc/elf64lppc/ configure
+
+export CPPFLAGS="${CPPFLAGS} -I${includedir} -I${includedir}/coin -I${includedir}/coin-or"
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+
+if [[ ${target} == *mingw* ]]; then
+    export LDFLAGS="-L${libdir}"
+    export LT_LDFLAGS="-no-undefined"
```

Comment:
https://docs.binarybuilder.org/dev/troubleshooting/#Libtool-refuses-to-build-shared-library-because-of-undefined-symbols

> Note that setting LDFLAGS=-no-undefined before ./configure would make this fail because it would run a command like cc -no-undefined conftest.c, which upsets the compiler).

Reply:
It's in the Makefile: https://github.com/coin-or/Bonmin/blob/e898c48a92057dd7acf424d6e10700b6ecdcc87a/Bonmin/src/Interfaces/Makefile.am#L52-L53
It doesn't resolve the problem locally, however. 

I've been trying with variations of 
```
./configure \
    --prefix=${prefix} \
    --build=${MACHTYPE} \
    --host=${target} \
    --with-pic \
    --disable-pkg-config \
    --disable-debug \
    --enable-shared \
    lt_cv_deplibs_check_method=pass_all \
    --with-asl-lib="-lasl -lipoptamplinterface" \
    --with-ipopt-lib="-lipopt" \
    --with-coindepend-lib="-lCbc -lCgl -lClp -lOsiClp -lOsi -lCoinUtils"
```
and I get more `__imp__` undefines. Is there anything special we need to do if Bonmin extends a lot of classes from Ipopt?